VaticanBAV/et/BAVet184.xml

```diff
@@ -0,0 +1,225 @@
+    
+    <msItem xml:id="ms_i1.7">
+        <locus from="141v" to="152v"/> 
+        <title ref="LIT1146Argano#Sunday"/>
+        <textLang mainLang="gez"/>    
+    </msItem>
+    
+</msItem>
+
+
+</msContents>
+
+<physDesc>
+  <objectDesc form="Codex">
+    <supportDesc>
+      <support>
+         <watermark/>
+         <material key="parchment"/>
+        </support>
+        <extent>
+          <measure unit="leaf" n="152">152</measure>
+          <measure unit="quire" n="19">19</measure>
+           <dimensions type="outer" unit="mm">
+              <height>125</height>
+              <width>110</width>
+           </dimensions>
+        </extent>
+        <collation>
+             <list>
+               <item xml:id="q1"><dim unit="leaf">8</dim><locus from="1r" to="8v"/></item>
+               <item xml:id="q2"><dim unit="leaf">8</dim><locus from="9r" to="16v"/></item>
+               <item xml:id="q3"><dim unit="leaf">8</dim><locus from="16r" to="24v"/></item>
+               <item xml:id="q4"><dim unit="leaf">8</dim><locus from="25r" to="32v"/></item>
+               <item xml:id="q5"><dim unit="leaf">8</dim><locus from="33r" to="40v"/></item>
+               <item xml:id="q6"><dim unit="leaf">8</dim><locus from="41r" to="48v"/></item>
+               <item xml:id="q7"><dim unit="leaf">8</dim><locus from="49r" to="56v"/></item>
+               <item xml:id="q8"><dim unit="leaf">8</dim><locus from="57r" to="64v"/></item>
+               <item xml:id="q9"><dim unit="leaf">8</dim><locus from="65r" to="72v"/></item>
+                 <item xml:id="q10"><dim unit="leaf">8</dim><locus from="73r" to="80v"/></item>
+                 <item xml:id="q11"><dim unit="leaf">8</dim><locus from="81r" to="88v"/></item>
+                 <item xml:id="q12"><dim unit="leaf">8</dim><locus from="89r" to="96v"/></item>
+                 <item xml:id="q13"><dim unit="leaf">7</dim><locus from="97r" to="103v"/></item>
+                 <item xml:id="q14"><dim unit="leaf">10</dim><locus from="104r" to="113v"/></item>
+                 <item xml:id="q15"><dim unit="leaf">7</dim><locus from="114r" to="120v"/></item>
+                 <item xml:id="q16"><dim unit="leaf">8</dim><locus from="121r" to="128v"/></item>
+                 <item xml:id="q17"><dim unit="leaf">8</dim><locus from="128r" to="136v"/></item>
+                 <item xml:id="q18"><dim unit="leaf">8</dim><locus from="137r" to="144v"/></item>
+                 <item xml:id="q19"><dim unit="leaf">8</dim><locus from="145r" to="152v"/></item>
+              </list>
+        </collation>
+    </supportDesc>
+    <layoutDesc>
+       <layout writtenLines="17">
+       </layout>
+    </layoutDesc>
+  </objectDesc>
+    
+    <handDesc>
+        <handNote xml:id="h1" script="Ethiopic">
+            <ab type="script"/>
+            <seg type="ink">Black, red</seg>
+            <seg type="rubrication">
+                Titles, nomina sacra, three of four lines at the beginning of the sections, cruces ansatae.
```

Comment:
It seems that, despite a good number of old MS records still have "nomina sacra", this is a wrong term, which rather refers to abbreviations. It should be better to use "holy names". 
I found an old (2019!) closed issue about this: https://github.com/BetaMasaheft/Documentation/issues/1144 in which it was hoped that all occurrences of "nomina sacra" would be corrected at some point.